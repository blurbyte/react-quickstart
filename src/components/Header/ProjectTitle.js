import styled from 'styled-components';

const ProjectTitle = styled.p`
  font-family: 'OpenSans Light', Arial, Helvetica, sans-serif;
  color: #908e89;
  font-size: 1.4rem;
  text-transform: uppercase;
  cursor: default;
`;

export default ProjectTitle;
