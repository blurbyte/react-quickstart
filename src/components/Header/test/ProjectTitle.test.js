import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import ProjectTitle from '../ProjectTitle';

describe('<ProjectTitle />', () => {
  it('should render correctly', () => {
    // component init
    const component = renderer.create(
      <ProjectTitle />
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
