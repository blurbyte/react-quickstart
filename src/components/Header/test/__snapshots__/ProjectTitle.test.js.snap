// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProjectTitle /> should render correctly 1`] = `
.c0 {
  font-family: 'OpenSans Light',Arial,Helvetica,sans-serif;
  color: #908e89;
  font-size: 1.4rem;
  text-transform: uppercase;
  cursor: default;
}

<p
  className="c0"
/>
`;
